### ISSUE TITLE
bug: Removable Pill Tooltips are not visible on keyboard focus

### ISSUE THREAD

**Issue by alanbsmith** [Workday/canvas-kit#3846]
## 🐛 Bug Report

Our Removable Pills can truncate

## Expected Behavior

Removable Pill Tooltips for truncated text should be available on hover AND keyboard focus.

## Actual Results

Removable Pill Tooltips for truncated text are ONLY available on hover.

## Link to repl or repo (highly encouraged)

[StackBlitz demo](https://stackblitz.com/edit/npxenxzt?file=src%2FDemo.tsx)

## Screenshots

Tooltip on hover

<img width="496" height="132" alt="Image" src="https://github.com/user-attachments/assets/f1037814-d670-4bed-8d2c-1c2aac09c459" />

No tooltip on focus

<img width="525" height="116" alt="Image" src="https://github.com/user-attachments/assets/c7e3bb78-0959-45f3-aff4-fda8dd7957a0" />
